Cargo.toml

```
@@ -126,7 +126,7 @@ rand = "=0.9.1"
 regex = "1"
 reqwest = { version = "0.12", default-features = false }
 rstest = "0.25.0"
-schemars = "0.8.21"
+schemars = "0.9.0"
```

```suggestion
schemars = "0.8.22"
```
schemars needs to be coordinated with apollo-federation-types